Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Stage 1: Builder
+FROM debian:buster-slim as builder
+
+SHELL ["/bin/bash", "-c"]
+WORKDIR /app
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV COMPOSER_ALLOW_SUPERUSER=1
+
+RUN apt-get update && apt-get install -y curl gnupg tzdata unzip git
+
+ENV TZ=America/New_York
```

Comment:
What timezone are we using for our other PHP apps? Set this to the same thing

Reply:
Removed timezone since we don't use it in any other Dockerfiles